### PR TITLE
Retail tech review - images and installation

### PR DESCRIPTION
**Pull request by aaannz**
* Move missing information about image building versions
  - Remove the mentions of SLE15 which is not yet released
  - Link images building from retail docu instead of providing info there
* Update installtion instructions for Retail
  - specify only retail related bits and the rest link from general config
  - update pictures not to show Beta/RC tags
  - add pictures for branch and system roles selection
  - add build host installation instructions
* Remove duplicate documentation of formulas
  - The formulas documentation is now in salt module and the text is
    exactly the same as in retail module. So remove the retail files
    to prevent confusion.
* Delta images - typo probably between sls file vs sles file
* Terminal deployment clarifications
  - Add link to image-sync formula
  - Clarify what happens if 32bit initrd is not used for 32bit machines
* Mass configuration clarification
  - add warnings about possible data loss
  - branch and terminal configurations are essentially the same and
    should have the same procedure
  - add data export prominent header
* PXE formula clarify it is saltboot kernel cmdline
* Saltboot formula enhancement
  - add EFI partition as first partition to allow terminals boot
    locally
  - add warning about local boot and bios/efi partition requirement
  - add info that applying highstate does not apply changes
* Specify smr in migration guide instead of susemgr
* Reorganize retail navigation
  - added top level section wrt terminal operations - deploying,
    naming, offline and limited network usage - which is one of the
    most important things done more than once
  - moved networking and arch overview under installation, but before
    setup where customer should have already clear idea about network
* Migration is no longer a tech preview

### CODE REVIEW

File: modules/administration/pages/image-management.adoc [hunk 1]
```diff
@@ -403,24 +404,40 @@ Disk space depends on the actual size of the image.
...
 
 
 To build all kinds of images with {productname}, create and configure a build host.
-OS Image build hosts are Salt clients running {sles}{nbsp}12 (SP3 or later) or 15.
+OS Image build hosts are Salt clients running on {sles}{nbsp}12 (SP3 or later) or {sles}{nbsp}11 SP4.
+
 This procedure will guide you through the initial configuration for a build host.
 
+[IMPORTANT]
+.Relation between image and build host OS version
```
Comment (Loquacity) [inline modules/administration/pages/image-management.adoc]:
```suggestion
```

File: modules/administration/pages/image-management.adoc [hunk 2]
```diff
@@ -403,24 +404,40 @@ Disk space depends on the actual size of the image.
...
+====
+Generally build host OS version should be the running the same OS version as the targeted image.
```
Comment (Loquacity) [inline modules/administration/pages/image-management.adoc]:
```suggestion
The operating system on the build host must match the operating system on the targeted image.
```

File: modules/administration/pages/image-management.adoc [hunk 3]
```diff
@@ -403,24 +404,40 @@ Disk space depends on the actual size of the image.
...
+
+- {sles}{nbsp}12 based images should be build on build host running {sles}{nbsp}12 OS version
```
Comment (Loquacity) [inline modules/administration/pages/image-management.adoc]:
```suggestion
For example, build {sles}{nbsp}12 based images on a build host running {sles}{nbsp}12 OS version.
```

File: modules/administration/pages/image-management.adoc [hunk 4]
```diff
@@ -403,24 +404,40 @@ Disk space depends on the actual size of the image.
...
+- {sles}{nbsp}11 based images should be build on build host running {sles}{nbsp}11 OS version
```
Comment (Loquacity) [inline modules/administration/pages/image-management.adoc]:
```suggestion
Build {sles}{nbsp}11 based images on a build host running {sles}{nbsp}11 OS version.
```

File: modules/administration/pages/image-management.adoc [hunk 5]
```diff
@@ -403,24 +404,40 @@ Disk space depends on the actual size of the image.
...
+====
+
+
 From the {productname} {webui}, perform these steps to configure a build host:
```
Comment (Loquacity) [inline modules/administration/pages/image-management.adoc]:
```suggestion
Configure the build host in the {productname} {webui}:
```

File: modules/administration/pages/image-management.adoc [hunk 6]
```diff
@@ -403,24 +404,40 @@ Disk space depends on the actual size of the image.
...
 
 . Select a client that will be designated as a build host from the menu:Systems[Overview] page.
 . From the menu:System Details[Properties] page, enable the [guimenu]``Add-on System Type:`` [guimenu]``OS Image Build Host`` and confirm with btn:[Update Properties].
```
Comment (Loquacity) [inline modules/administration/pages/image-management.adoc]:
```suggestion
. Navigate to menu:System Details[Properties] page, enable the [guimenu]``Add-on System Type:`` [guimenu]``OS Image Build Host`` and confirm with btn:[Update Properties].
```

File: modules/administration/pages/image-management.adoc [hunk 7]
```diff
@@ -403,24 +404,40 @@ Disk space depends on the actual size of the image.
...
 +
 image::os-image-build-host.png[scaledwidth=80%]
 +
-. From the menu:System Details[Software > Software Channels] page, enable ``SLE-Manager-Tools12-Pool`` and ``SLE-Manager-Tools12-Updates`` (or a later version).
+. From the menu:System Details[Software > Software Channels] page, enable {productname} Client tools (e.g. ``SLE-Manager-Tools12-Pool`` and ``SLE-Manager-Tools12-Updates``).
```
Comment (Loquacity) [inline modules/administration/pages/image-management.adoc]:
```suggestion
. Navigate to menu:System Details[Software > Software Channels] page, enable {productname} Client tools (for example: ``SLE-Manager-Tools12-Pool`` and ``SLE-Manager-Tools12-Updates``).
```

File: modules/retail/pages/retail-components.adoc [hunk 8]
```diff
@@ -17,10 +17,8 @@ This can include database entries of registered systems, Salt pillar data for im
 
 Build hosts can be arbitrary servers or virtual machines.
 They are used to securely build operating system images.
-// REMARK: Check version number
-Build hosts should be based on {sles}{nbsp}12 SP3 or SP4.
-
 
+For more information on build hosts, see documentation of OS Images building feature (xref:administration:image-management.adoc[])
```
Comment (Loquacity) [inline modules/retail/pages/retail-components.adoc]:
```suggestion
For more information on build hosts, see xref:administration:image-management.adoc[].
```

File: modules/retail/pages/retail-install-unified.adoc [hunk 9]
```diff
@@ -3,13 +3,12 @@
 
 {smr} 4.0 is a {suse} base product.
 This section describes how to install {smr} from {sles} installation media with the Unified Installer.
-It assumes you have already registered the {susemgr} product with the {scc} and have a registration code.
 
-For information on registering with {scc}, or obtaining installation media, see xref:installation:general-requirements.adoc[].
+Before you begin installation, ensure you are familiar with xref:installation:general-requirements.adoc[general requirements].
```
Comment (Loquacity) [inline modules/retail/pages/retail-install-unified.adoc]:
```suggestion
Before you begin the installation, check that your environment meets the requirements at xref:installation:general-requirements.adoc[].
```

File: modules/retail/pages/retail-install-unified.adoc [hunk 10]
```diff
@@ -3,13 +3,12 @@
...
 
+== Installing {smr}
 
-
-.Procedure: Installing {susemgr} Server from a DVD Image
+.Procedure: Installing {smr} Server from DVD Image
```
Comment (Loquacity) [inline modules/retail/pages/retail-install-unified.adoc]:
```suggestion
.Procedure: Installing {smr} Server from a DVD Image
```

File: modules/retail/pages/retail-install-unified.adoc [hunk 11]
```diff
@@ -60,13 +67,29 @@ In case of trouble, you might need to adjust the boot order in the BIOS.
...
-When the installation procedure has finished, you can check that all the required modules are added.
-Use the [command]``SUSEConnect --status-text`` command at a command prompt.
-For {smr}, the expected modules are:
-
-* {sles} Basesystem Module
-* Python 2 Module
-* Server Applications Module
-* SUSE Manager Proxy Module
-* SUSE Manager Retail Branch Server Module
+.Procedure: Configuration and registering of the Branch Server
```
Comment (Loquacity) [inline modules/retail/pages/retail-install-unified.adoc]:
```suggestion
.Procedure: Configuring and Registering the Branch Server
```

File: modules/retail/pages/retail-install-unified.adoc [hunk 12]
```diff
@@ -60,13 +67,29 @@ In case of trouble, you might need to adjust the boot order in the BIOS.
...
+. Create an activation key based on the [systemitem]``SLE-Product-SUSE-Manager-Retail-Branch-Server-4.0-Pool`` base channel.
+For more information about activation keys, see xref:client-configuration:clients-and-activation-keys.adoc[Creating Activation Key].
+. From the [guimenu]``Child Channels`` listing select the recommended
```
Comment (Loquacity) [inline modules/retail/pages/retail-install-unified.adoc]:
```suggestion
. In the [guimenu]``Child Channels`` listing, select the recommended channels by clicking the ``include recommended`` icon:
```

File: modules/retail/pages/retail-install-unified.adoc [hunk 13]
```diff
@@ -60,13 +67,29 @@ In case of trouble, you might need to adjust the boot order in the BIOS.
...
+channels by clicking the ``include recommended`` icon:
```
Comment (Loquacity) [inline modules/retail/pages/retail-install-unified.adoc]:
```suggestion
```

File: modules/retail/pages/retail-install-unified.adoc [hunk 14]
```diff
@@ -60,13 +67,29 @@ In case of trouble, you might need to adjust the boot order in the BIOS.
...
++
+* SLE-Module-Basesystem15-SP1-Pool for x86_64 SMRBS 4.0
+* SLE-Module-Basesystem15-SP1-Updates for x86_64 SMRBS 4.0
+* SLE-Module-Server-Applications15-SP1-Pool for x86_64 SMRBS 4.0
+* SLE-Module-Server-Applications15-SP1-Updates for x86_64 SMRBS 4.0
+* SLE-Product-SUSE-Manager-Retail-Branch-Server-4.0-Updates for x86_64
+. With this activation key created, continue with step 3 of the {susemgrproxy} registration at xref:installation:proxy-registration[].
```
Comment (Loquacity) [inline modules/retail/pages/retail-install-unified.adoc]: I can see this going awry. Can we organise this differently to avoid the "step 3" bit?

File: modules/retail/pages/retail-install-unified.adoc [hunk 15]
```diff
@@ -60,13 +67,29 @@ In case of trouble, you might need to adjust the boot order in the BIOS.
...
+
+
+[WARNING]
+.{smr} salt requirement
```
Comment (Loquacity) [inline modules/retail/pages/retail-install-unified.adoc]:
```suggestion
```

File: modules/retail/pages/retail-install-unified.adoc [hunk 16]
```diff
@@ -60,13 +67,29 @@ In case of trouble, you might need to adjust the boot order in the BIOS.
...
+====
+Do not configure the branch server as a traditionally managed proxy. Salt client approach must be used.
```
Comment (Loquacity) [inline modules/retail/pages/retail-install-unified.adoc]:
```suggestion
Do not configure the branch server as a traditionally managed proxy. 
The branch server must be configured as a Salt client.
```

File: modules/retail/pages/retail-install-unified.adoc [hunk 17]
```diff
@@ -60,13 +67,29 @@ In case of trouble, you might need to adjust the boot order in the BIOS.
...
+====
+
+[NOTE]
+.Cobbler and {smr}
```
Comment (Loquacity) [inline modules/retail/pages/retail-install-unified.adoc]:
```suggestion
```

File: modules/retail/pages/retail-install.adoc [hunk 18]
```diff
@@ -1,9 +1,9 @@
 [[retail-install]]
 = Installation
 
-{smr} can be installed in various ways depending on individual needs.
+{smr} and {smr} Branch Server are installed using {sles} Unified Installer. Next section describes the installation process.
```
Comment (Loquacity) [inline modules/retail/pages/retail-install.adoc]:
```suggestion
{smr} and {smr} Branch Server are installed using the {sles} Unified Installer. 
```

File: modules/retail/pages/retail-install-unified.adoc (hunk 12 above)
Comment (aaannz) [reply modules/retail/pages/retail-install-unified.adoc]: I'll change it in my changes, but this is copy paste (with modified channel list) from https://github.com/SUSE/doc-susemanager/blob/master/modules/installation/pages/proxy-registration.adoc#suse-manager-proxy-registration

File: modules/retail/pages/retail-install-unified.adoc (hunk 14 above)
Comment (aaannz) [reply modules/retail/pages/retail-install-unified.adoc]: I can copy paste the rest of proxy registration procedure. Branch Server is essentially Proxy, just with different channels. The configuration is exactly the same.

Comment (Loquacity) [reply modules/retail/pages/retail-install-unified.adoc]: Hrm, I'm not a fan of duplication, either. Can we split the first two steps off for branch server and proxy, then make the remainder for both?

Comment (Loquacity) [reply modules/retail/pages/retail-install-unified.adoc]: Or, list the differences for each in one procedure

File: modules/retail/pages/retail-install-unified.adoc (hunk 12 above)
Comment (Loquacity) [reply modules/retail/pages/retail-install-unified.adoc]: Yeah, even tech writers write badly sometimes ;)

**Comment by Loquacity** [comment]
We'll want to squash these before merge.

File: modules/administration/pages/image-management.adoc [hunk 19]
```diff
@@ -403,24 +404,39 @@ Disk space depends on the actual size of the image.
 For more information, see the documentation of the underlying system.
 ====
 
+[WARNING]
+====
+The build host must be a Salt client.
+Do not install the build host as a traditionally managed client.
```
Comment (keichwa) [inline modules/administration/pages/image-management.adoc]:
```suggestion
Do not install the build host as a traditional client.
```
Now, we call these beasts just "traditional client". Or is the longer version also fine, @Loquacity ?

File: modules/administration/pages/image-management.adoc [hunk 20]
```diff
@@ -403,24 +404,39 @@ Disk space depends on the actual size of the image.
...
+For example, build {sles}{nbsp}12 based images on a build host running {sles}{nbsp}12 OS version.
+Build {sles}{nbsp}11 based images on a build host running {sles}{nbsp}11 OS version.
+====
+
+
+Configure the build host in the {productname} {webui}:
 
 . Select a client that will be designated as a build host from the menu:Systems[Overview] page.
-. From the menu:System Details[Properties] page, enable the [guimenu]``Add-on System Type:`` [guimenu]``OS Image Build Host`` and confirm with btn:[Update Properties].
+. Navigate to menu:System Details[Properties] page, enable the [guimenu]``Add-on System Type:`` [guimenu]``OS Image Build Host`` and confirm with btn:[Update Properties].
```
Comment (keichwa) [inline modules/administration/pages/image-management.adoc]:
```suggestion
. Navigate to the menu:System Details[Properties] tab, enable the [guimenu]``Add-on System Type`` [guimenu]``OS Image Build Host``.
Confirm with btn:[Update Properties].
```
Even if a label is finished with ":", I'd not add the colon if you reference the name of the label in the running text.

File: modules/administration/pages/image-management.adoc [hunk 21]
```diff
@@ -403,24 +404,39 @@ Disk space depends on the actual size of the image.
...
 +
 image::os-image-build-host.png[scaledwidth=80%]
 +
-. From the menu:System Details[Software > Software Channels] page, enable ``SLE-Manager-Tools12-Pool`` and ``SLE-Manager-Tools12-Updates`` (or a later version).
+. Navigate to menu:System Details[Software > Software Channels] page, enable {productname} Client tools (for example: ``SLE-Manager-Tools12-Pool`` and ``SLE-Manager-Tools12-Updates``).
```
Comment (keichwa) [inline modules/administration/pages/image-management.adoc]:
```suggestion
. Navigate to the menu:System Details[Software > Software Channels] tab, enable {productname} Client tools (for example: ``SLE-Manager-Tools12-Pool`` and ``SLE-Manager-Tools12-Updates``).
```

File: modules/retail/nav-retail.adoc [hunk 22]
```diff
@@ -3,33 +3,24 @@ ifndef::backend-pdf[]
...
-//*** xref:retail-formula-pxe.adoc[PXE Formula]
-//*** xref:retail-formula-saltboot.adoc[Saltboot Formula]
-//*** xref:retail-formula-tftpd.adoc[TFTPd Formula]
-//*** xref:retail-formula-vsftpd.adoc[VsFTPd Formula]
 // Admin
 ** xref:retail-admin.adoc[Administration]
-*** xref:retail-delta-images.adoc[Delta Images]
-*** xref:retail-terminal-names.adoc[Terminal Names]
 *** xref:retail-mass-config.adoc[Mass Configuration]
 *** xref:retail-mass-config-yaml.adoc[Mass Config YAML Example]
```
Comment (keichwa) [inline modules/retail/nav-retail.adoc]:
```suggestion
*** xref:retail-mass-config-yaml.adoc[Mass Configuration YAML Example]
```
Drive-by edit.

File: modules/retail/pages/retail-deploy-terminals.adoc [hunk 23]
```diff
@@ -106,7 +111,7 @@ legacy-6.0.0:
 
 In this example, the [systemitem]``legacy-6.0.0`` boot image is 32-bit.
 
-You can set the default boot image in the ``Image Synchronization`` formula on the branch server, by adding the chosen boot image name to the [guimenu]``Default boot image`` field.
+You can set the default boot image in the ``Image Synchronization`` formula on the branch server, by adding the chosen boot image name to the [guimenu]``Default boot image`` field. For more information about ``Image Synchronization`` formula, see xref:salt:formula-imagesync.adoc[].
```
Comment (keichwa) [inline modules/retail/pages/retail-deploy-terminals.adoc]:
```suggestion
You can set the default boot image in the ``Image Synchronization`` formula on the branch server, by adding the chosen boot image name to the [guimenu]``Default boot image`` field.
For more information about ``Image Synchronization`` formula, see xref:salt:formula-imagesync.adoc[].
```

File: modules/retail/pages/retail-install-unified.adoc [hunk 24]
```diff
@@ -60,13 +67,42 @@ In case of trouble, you might need to adjust the boot order in the BIOS.
 Review the settings and click btn:[Install].
 
 
+.Procedure: Configuring and Registering the Branch Server
+. Create an activation key based on the [systemitem]``SLE-Product-SUSE-Manager-Retail-Branch-Server-4.0-Pool`` base channel.
+For more information about activation keys, see xref:client-configuration:clients-and-activation-keys.adoc[Creating Activation Key].
```
Comment (keichwa) [inline modules/retail/pages/retail-install-unified.adoc]:
```suggestion
For more information about activation keys, see xref:client-configuration:clients-and-activation-keys.adoc[].
```
We want to derive this text from the heading of the xref'ed file.

File: modules/retail/pages/retail-install-unified.adoc [hunk 25]
```diff
@@ -60,13 +67,42 @@ In case of trouble, you might need to adjust the boot order in the BIOS.
...
+
+[NOTE]
+====
+Cobbler is not currently supported within {smr} environments.
+If you intend to use your installation with {smr} formulas, do not configure Cobbler on the {smr} branch server.
+====
+
+== Installing {susemgr} Build Host
+
+Build hosts are regular {sles} installations registered to {susemgr} as salt minions. For more information how to install and register salt minions to {susemgr} see xref:client-configuration:registration-overview.adoc[].
```
Comment (keichwa) [inline modules/retail/pages/retail-install-unified.adoc]:
```suggestion
Build hosts are regular {sles} installations registered to {susemgr} as salt minions.
For more information how to install and register salt minions to {susemgr}, see xref:client-configuration:registration-overview.adoc[].
```

File: modules/retail/pages/retail-mass-config.adoc [hunk 26]
```diff
@@ -13,6 +13,14 @@ If you are working in an environment with many branch servers, you might find it
 
 // https://gitlab.suse.de/SLEPOS/SUMA_Retail/tree/master/python-susemanager-retail
 
+[WARNING]
+====
+Before working with mass configuration tool, backup existing branch servers configuration.
```
Comment (keichwa) [inline modules/retail/pages/retail-mass-config.adoc]:
```suggestion
Before working with the mass configuration tool, back up the existing branch servers configuration.
```

File: modules/retail/pages/retail-mass-config.adoc [hunk 27]
```diff
@@ -13,6 +13,14 @@ If you are working in an environment with many branch servers, you might find it
...
+
+Mass configuration tool overwrites existing configuration with data specified in provided YAML file.
```
Comment (keichwa) [inline modules/retail/pages/retail-mass-config.adoc]:
```suggestion
The mass configuration tool overwrites the existing configuration with data specified in the provided YAML file.
```

File: modules/retail/pages/retail-mass-config.adoc [hunk 28]
```diff
@@ -13,6 +13,14 @@ If you are working in an environment with many branch servers, you might find it
...
+
+Mass configuration tool does not support all possible formula configurations. Always make sure on small sample, that mass configuration tool can configure systems as expected.
```
Comment (keichwa) [inline modules/retail/pages/retail-mass-config.adoc]:
```suggestion
The mass configuration tool does not support all possible formula configurations.
Always make sure on a small sample, that the mass configuration tool can configure systems as expected.
```

File: modules/retail/pages/retail-mass-config.adoc [hunk 29]
```diff
@@ -31,6 +39,11 @@ retail_yaml --from-yaml filename.yaml
 ----
 +
 See the [command]``retail_yaml --help`` output for additional options.
+. In the {susemgr} {webui}, check that your systems are listed and displaying correctly, and the formulas you require are available.
+. Create activation keys for each of your branch servers, connect them using bootstrap, and configure them as proxy servers.
+For further information, xref:retail:retail-install-unified.adoc[].
```
Comment (keichwa) [inline modules/retail/pages/retail-mass-config.adoc]:
```suggestion
For more information, see xref:retail:retail-install-unified.adoc[].
```

File: modules/retail/pages/retail-mass-config.adoc [hunk 30]
```diff
@@ -72,19 +85,21 @@ retail_yaml --from-yaml filename.yaml
 See the [command]``retail_yaml --help`` output for additional options.
 . In the {susemgr} {webui}, check that your systems are listed and displaying correctly, and the formulas you require are available.
 . Create activation keys for each of your branch servers, connect them using bootstrap, and configure them as proxy servers.
-// FIXME: look up exact refs
-For further information on these steps, see the {susemgr} documentation.
-. In the [guimenu]``States`` tab, click btn:[Apply Highstate] to deploy your configuration changes.
+For further information, xref:retail:retail-install-unified.adoc[].
```
Comment (keichwa) [inline modules/retail/pages/retail-mass-config.adoc]:
```suggestion
For more information, see xref:retail:retail-install-unified.adoc[].
```

File: modules/retail/pages/retail-mass-config.adoc [hunk 31]
```diff
@@ -72,19 +85,21 @@ retail_yaml --from-yaml filename.yaml
...
+. In the [guimenu]``States`` tab, click btn:[Apply Highstate] to deploy your configuration changes for each branch server.
 . Connect your terminals according to your infrastructure plan.
 
 If you need to change your configuration, you can edit the YAML file at any time, and use the [command]``retail_yaml --from-yaml`` command to upload the new configuration.
 
+
+== Exporting Configuration to Mass Configuration File
```
Comment (keichwa) [inline modules/retail/pages/retail-mass-config.adoc]:
```suggestion
== Export Configuration to Mass Configuration File
```

File: modules/retail/pages/retail-migration-slepostosuma.adoc [hunk 32]
```diff
@@ -15,8 +15,8 @@ Importing and deploying the converted data can also be done in one or multiple s
 
 In this procedure, you create a single data dump in an XML file, convert it to YAML, and migrate it to the new infrastructure all at once.
 
-. Install a {susemgr} server {productnumber} (with the Retail pattern).
-For more information, see xref:retail:retail-install.adoc[].
+. Install a {smr} server {productnumber}
```
Comment (keichwa) [inline modules/retail/pages/retail-migration-slepostosuma.adoc]:
```suggestion
. Install a {smr} server {productnumber}.
```

File: modules/retail/pages/retail-migration-slepostosuma.adoc [hunk 33]
```diff
@@ -70,8 +70,8 @@ After all the branches are migrated, shutdown and remove the old SLEPOS Admin Se
 
 In this procedure, you migrate the SLEPOS infrastructure and the branches one by one, first exporting and then importing.
 
-. Install a {susemgr} server {productnumber} (with a [literal]``SUSE Manager for Retail Server`` [guimenu]``System Role``).
-For more information, see xref:retail:retail-install.adoc[].
+. Install a {smr} server {productnumber}
```
Comment (keichwa) [inline modules/retail/pages/retail-migration-slepostosuma.adoc]:
```suggestion
. Install a {smr} server {productnumber}.
```

File: modules/salt/pages/formula-saltboot.adoc [hunk 34]
```diff
@@ -9,6 +9,11 @@ The Saltboot formula is meant to be used as a group formula.
 Enable and configure Saltboot formula for hardware type groups.
 ====
 
+[IMPORTANT]
+====
+To apply changes to a terminal, terminal needs to be restarted. Applying highstate does not have any effect on running terminals.
```
Comment (keichwa) [inline modules/salt/pages/formula-saltboot.adoc]:
```suggestion
To apply changes to a terminal, the terminal needs to be restarted.
Applying highstate does not have any effect on running terminals.
```

File: modules/retail/pages/retail-mass-config.adoc (hunk 31 above)
Comment (aaannz) [reply modules/retail/pages/retail-mass-config.adoc]: I fixed this in my changes, but I am little confused here. Because in another file, @Loquacity suggested to use present continuous tense. What tense should be used in headers?

**Comment by aaannz** [comment]
> We'll want to squash these before merge.

Yes, I'll squash the commits once all questions are cleared. I hope that seeing individual commits is easier for you to see the changes.

File: modules/retail/pages/retail-install-unified.adoc (hunk 14 above)
Comment (aaannz) [reply modules/retail/pages/retail-install-unified.adoc]: The difference to proxy configuration is that different channels are used for branch servers.

So I can start the procedure with:

> Branch server is based on {susemgrproxy} and the registration procedure is almost the same.
> Use this procedure to create activation key for the branch server. Then use created activation key in usual {susemgrproxy} registration procedure as described at xref:installation:proxy-registration[].

Or finish the procedure with:

> Use this activation key in {susemgrproxy} registration as instructed at xref:installation:proxy-registration[].

File: modules/administration/pages/image-management.adoc (hunk 19 above)
Comment (Loquacity) [reply modules/administration/pages/image-management.adoc]: I prefer shorter.

File: modules/retail/pages/retail-mass-config.adoc (hunk 31 above)
Comment (Loquacity) [reply modules/retail/pages/retail-mass-config.adoc]: Use simple present for headings, and present continuous for procedures. We're a bit more specific than that, though, by looking at the verb type instead of the tenses: use bare infinitive verb form for the section heading, gerund verb form for the procedure title (cf:  https://github.com/SUSE/doc-susemanager/wiki/wug#headings and https://github.com/SUSE/doc-susemanager/wiki/wug#procedure-titles I will update this with the tense info, though, since you made me notice it's missing!). 

It's entirely possible I did the wrong thing somewhere along the line though, I *am* human :wink:

Comment (aaannz) [reply modules/retail/pages/retail-mass-config.adoc]: Thanks for explanation and the links to the wiki, I completely missed that one. :)

**Comment by aaannz** [comment]
All commits squashed. Ready for final ack and merge (which I can't do).
